trajan/readers/omb.py

```diff
@@ -282,6 +285,8 @@ def read_omb_csv(path_in: Path,
                     (trajectory, obs_waves_imu, frequencies_waves_imu)),
                 attrs={
                     "_FillValue": "NaN",
+                    "long_name": "sea_surface_wave_elevation_energy_spectrum",
```

Comment:
I think there exists a standard name for this. It should probably be the same for the processed and the non-processed.